prime-router/src/main/kotlin/fhirengine/engine/CustomTranslationFunctions.kt

```diff
@@ -28,38 +28,36 @@ class CustomTranslationFunctions(
         element: ConverterSchemaElement?,
         constantSubstitutor: ConstantSubstitutor?,
     ): String {
-        if (appContext?.config is HL7TranslationConfig) {
-            val receiver = appContext.config.receiver
-            val config = appContext.config.hl7Configuration
-            var dateTimeFormat = receiver?.dateTimeFormat
+        check(appContext != null)
```

Comment:
Probably would be cleaner to change this method to make appContext not nullable?